terraso_backend/config/settings.py

```diff
@@ -221,6 +221,14 @@
     WEB_CLIENT_URL = f"{WEB_CLIENT_URL}:{WEB_CLIENT_PORT}"
 
 
+FILE_SERVER_DOMAIN = config("FILE_SERVER_DOMAIN", default="")
+FILE_SERVER_PORT = config("FILE_SERVER_PORT", default=443)
+FILE_SERVER_PROTOCOL = config("FILE_SERVER_ROTOCOL", default="https")
+
+FILE_SERVER_URL = f"{FILE_SERVER_PROTOCOL}://{FILE_SERVER_DOMAIN}"
+if FILE_SERVER_PORT != 443:
+    FILE_SERVER_URL = f"{FILE_SERVER_URL}:{FILE_SERVER_PORT}"
+
```

Comment:
* for FILE_SERVER bits, we only need a URL, since AWS is always `https` and always port `443`.
* we already have the variable we need in `DATA_ENTRY_FILE_S3_BUCKET`

Reply:
Please write descriptive commit messages.